resources/deployments/controller/controller_deployments.go

```diff
@@ -341,6 +369,18 @@ func (d *DeploymentsController) LookupDeployment(w rest.ResponseWriter, r *rest.
 		return
 	}
 
+	for idx, dep := range deps {
```

Comment:
This is business logic, should be part of business model. Controller handles protocol logic.